snippets/PowerShell.json

```diff
@@ -1082,5 +1082,65 @@
             "}"
         ],
         "description": "IArgumentCompleter implementation class definition"
+    },
+    "Requires Assembly": {
+        "prefix": "requires-assembly",
+        "body": "#Requires -Assembly '${1:${TM_SELECTED_TEXT:fully-qualified-name}}'",
+        "description": "Requires an assembly (by name) in order to execute the containing script file."
+    },
+    "Requires Assembly Path": {
+        "prefix": "requires-assembly-path",
+        "body": "#Requires -Assembly ${0:${TM_SELECTED_TEXT:path/to/assembly.dll}}",
+        "description": "Requires an assembly (by relative or absolute path) in order to execute the containing script file."
+    },
+    "Requires Assembly Version": {
+        "prefix": "requires-assembly-version",
+        "body": "#Requires -Assembly '${1:${TM_SELECTED_TEXT:fully-qualified-name}}, Version=${2:1.0.0.0}'",
+        "description": "Requires an assembly (by name and minimum version) in order to execute the containing script file."
+    },
+    "Requires Module": {
+        "prefix": "requires-module",
+        "body": "#Requires -Module ${0:${TM_SELECTED_TEXT:fully-qualified-name}}",
+        "description": "Requires a module (by name) in order to execute the containing script file."
+    },
+    "Requires Module RequiredVersion": {
```

Comment:
I think this one and line 1111 got mixed up.

Reply:
Fixed by 7546e2a